packages/desktop-client/e2e/rules.mobile.test.ts

```diff
@@ -52,24 +52,22 @@ test.describe('Mobile Rules', () => {
     await expect(page).toMatchThemeScreenshots();
   });
 
-  test('clicking add button opens rule creation modal', async () => {
+  test('clicking add button opens rule creation form', async () => {
     await rulesPage.clickAddRule();
 
-    // Check that edit rule modal is opened
-    const modal = page.getByRole('dialog');
-    await expect(modal).toBeVisible();
     await expect(page).toMatchThemeScreenshots();
   });
 
-  test('clicking on a rule opens edit modal', async () => {
+  test('clicking on a rule opens edit form', async () => {
```

Comment:
The snapshots with the old name will need to be manually deleted

Reply:
Good catch: https://github.com/actualbudget/actual/pull/5516/commits/662d6e5a33d20126a19b7aef4c492e8940a47f27